modin/config/pubsub.py

```diff
@@ -75,6 +75,22 @@ class ExactStr(str):
         or (isinstance(value, str) and value.strip().isdigit()),
         help="an integer value",
     ),
+    dict: TypeDescriptor(
+        decode=lambda value: {
+            key.strip(): int(val) if val.strip().isdigit() else val.strip()
+            for key_value in value.split(",")
+            for key, val in [key_value.split("=")]
+        },
+        normalize=lambda value: value
```

Comment:
not sure what the difference between `decode` and `normalize`, isn't it's just the same?

Reply:
So in the case of `type=dict` we expect normalize's parameter to be either string or dict and so have to duplicate the logic of decode?

Reply:
No, in the case of `normalize` for dictionary I think we'd expect incoming value to be of type `dict` only (much like we do for `bool` or `int`).
It's `verify` that's going to be somewhat a duplicate I think...

Reply:
Or you can just make a regex to validate :)

Reply:
If `verify` "approves" both: strings and dictionaries, but `normalize` takes only dicts, then we're risking to pass a raw string to `normalize` and get an error there. So I consider keeping `normalize` as it is.